lib/stdlib/putchar.c

```diff
@@ -15,6 +16,8 @@
 int putchar(int c)
 {
 	int res;
+	if (COREBOOT)
+		coreboot_cbmc_putc(c);
```

Comment:
Hooking into the stdlib putchar is not appropriate. This should be implemented within the existing console framework.